### PR TITLE
Feature/load input schema as stream

### PR DESCRIPTION
**Pull request by hesserp**
Load input schema as stream. Otherwise resource files cannot be found when the project is packed into a .jar

### CODE REVIEW

File: src/test/scala/org/zalando/spark/jsonschema/SparkTestEnv.scala [hunk 1]
```diff
@@ -15,4 +17,13 @@ object SparkTestEnv {
       .getOrCreate()
   }
 
+  def getTestResourceContent(relativePath: String): String = {
+    val inputStream = getClass.getResourceAsStream(relativePath)
+    try Source.fromInputStream(inputStream).mkString
```
Comment (ovolynets) [inline src/test/scala/org/zalando/spark/jsonschema/SparkTestEnv.scala]: I think the style should be with the curly braces:
```scala
try {
  ...
} catch {
  ...
}
```

File: src/test/scala/org/zalando/spark/jsonschema/SparkTestEnv.scala [hunk 2]
```diff
@@ -15,4 +17,13 @@ object SparkTestEnv {
...
+    catch {
+      case _: NullPointerException =>
+        throw new NullPointerException(s"No content found in $relativePath")
```
Comment (ovolynets) [inline src/test/scala/org/zalando/spark/jsonschema/SparkTestEnv.scala]: NullPointerException is thrown by the system. I'd suggest to use RuntimeException or IllegalArgumentException, or something similar.

**Comment by tammok** [comment]
:+1:

**Comment by hesserp** [comment]
:+1:

**Pull request merged**